engine/gamesys/src/gamesys/components/comp_sprite.cpp

```diff
@@ -1025,10 +1025,14 @@ namespace dmGameSystem
         return !data->m_UsesGeometries || data->m_Geometries[0]->m_TrimMode == dmGameSystemDDF::SPRITE_TRIM_MODE_OFF;
     }
 
-    static void ResolveUVDataFromGeometry(TexturesData* data, dmArray<float>* scratch_uvs, uint16_t flip_horizontal, uint16_t flip_vertical)
+    static void ResolvePositionAndUVDataFromGeometry(TexturesData* data, dmArray<float>* scratch_pos, dmArray<float>* scratch_uvs,
```

Comment:
Adding the "position" array as output here, so that we only have to do the "loop logic" once, to simplify things.